src/database/migrations/20220920084207-create-redirects.js

```diff
@@ -0,0 +1,39 @@
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("redirections", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      launch_id: {
+        allowNull: false,
+        type: Sequelize.INTEGER,
+      },
```

Comment:
I think this should also have the `references` key as well, same as above.

Reply:
fixed: 6721422